ApplyToBecomeInternal/ApplyToBecome.Data/Models/AdvisoryBoardDecision/AdvisoryBoardDecisions.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace ApplyToBecome.Data.Models.AdvisoryBoardDecision
+{
+	public enum AdvisoryBoardDecisions
+	{
+		Approved = 0,
```

Comment:
Shouldn't we have 0 for a non-decided project decision. Having 0 assigned to approved will make every project decision approved by default. Is that ok?

Reply:
Again, I've taken this model from the academisation api. @MartinWheelerMT what are your thoughts on this?

Reply:
I'm always a fan of having default / null in enums personally.  In this case there isn't a record until the decision has been created and you have to select an option when recording the decision.  There isn't a state where a project has recorded a decision but no has not made a choice, to be able to record a decision it will have to be Approved, Denied or Deferred

Reply:
Ok, I'll make the enum nullable in the PageModel. So the user has to select the radio button before continuing.
I had it pre-selected because I wasn't quite ready to do validation messages. But as we've had this discussion I'm happy to change it.